packages/api/prisma/migrations/20230328215901_add_languages/migration.sql

```diff
@@ -0,0 +1,19 @@
+-- CreateExtension
+CREATE EXTENSION IF NOT EXISTS "pgcrypto";
+
+CREATE OR REPLACE FUNCTION generate_ulid() RETURNS uuid
+    AS $$
+        SELECT (lpad(to_hex(floor(extract(epoch FROM clock_timestamp()) * 1000)::bigint), 12, '0') || encode(gen_random_bytes(10), 'hex'))::uuid;
+    $$ LANGUAGE SQL;
```

Comment:
I swiped this from that [same page](https://blog.daveallie.com/ulid-primary-keys)